### ISSUE TITLE
accessibility review of Alerts demo

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#629]
When a demo is implemented showing inline notifications and toast notifications, an a11y review of these working components should be completed to catch any potential issues related to accessibility.

**Comment by jgiardino** [comment]
Closing this and opening in pf-react instead: https://github.com/patternfly/patternfly-react/issues/543
